app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the 
+<a href="http://www.railstutorial.org/help">Rails Tutorial help page</a>.
+To get help on this sample app, see the 
+<a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
```

Comment:
format your html
tab 2 space if it inside p

Reply:
Done